lib/datapacksexpand.js

```diff
@@ -438,10 +447,10 @@ DataPacksExpand.prototype.getNestedDataPackReferences = function(currentData, pa
 
         if (Array.isArray(currentData[primaryField])) {
             for (var nestedMore of currentData[primaryField]) {
-                this.getNestedDataPackReferences(nestedMore, parentKeys, splitFields.join('.'), referenceType);
+                this.getNestedDataPackReferences(jobInfo, nestedMore, parentKeys, splitFields.join('.'), referenceType);
```

Comment:
Since `async` was added to this method, you'll need to add `await` to these `getNestedDataPackReferences` calls.

Reply:
Updated